Sources/StreamVideo/Utils/SystemEnvironment+XStreamClient.swift

```diff
@@ -61,7 +77,7 @@ extension SystemEnvironment {
 
     private static var os: String {
         #if os(iOS)
-        return "iOS"
+        return isSimulator ? "iOS-simulator" : "iOS"
```

Comment:
do we need this distinction? I think it might mess up nessy stats.

Reply:
I think it may help identify performance issues that aren't happening on both the real device and the simulators but that's ok if it messes with the stats.